apix-rest-v1/src/main/java/eu/xenit/apix/rest/v1/nodes/NodesWebscript1.java

```diff
@@ -963,6 +963,14 @@ private void writeNotAuthorizedResponse(AccessDeniedException exception, WebScri
         writeJsonResponse(response, "Not authorised to execute this operation");
     }
 
+    @ExceptionHandler(FileExistsException.class)
+    private void writeFileExistsResponse(FileExistsException fileExistsException, WebScriptResponse response) throws IOException {
+        String message = String.format("A file with name %s already exists at %s", fileExistsException.getName(), fileExistsException.getParentNodeRef());
```

Comment:
Do you know why it is get**parent**noderef? I know there is no accessor for it, but I assume the duplicate file should have its own noderef, no?

Reply:
The called method handles all 3 functions (create, copy, move). The copy and move both have a `copyFrom` argument, which we could probably access and print. Ill try adding that. Obviously the create does not.
I suppose the idea behind the exception (from the alfresco code) is that it doesnt really matter where the node came from, so it only includes the parent? ill see if i can our code.